Makefile

```diff
@@ -145,7 +145,10 @@ controller-gen: ## Download controller-gen locally if necessary
 .PHONY: kustomize
 KUSTOMIZE = $(shell pwd)/bin/kustomize
 kustomize: ## Download kustomize locally if necessary
-	$(call go-get-tool,$(KUSTOMIZE),sigs.k8s.io/kustomize/kustomize/v4@v4.0.1)
+	rm -f ./kustomize
```

Comment:
```## Download kustomize locally if necessary```

I think we should check if there is a binary and install only if it's missing

Reply:
Good point. Fixed!